routes/api/auth/index.js

```diff
@@ -38,12 +38,21 @@ router.post('/', async (req, res) => {
     if (!valid) {
         return res.status(400).json({error: 'Invalid password'});
     }
+    
+    const rememberMe = req.body.rememberMe;
+    
+    if(rememberMe){
+        // Sets cookie length to 1 year
+        req.session.cookie.maxAge = 365 * 24 * 60 * 60 * 1000;
+        req.session.rolling = true;
```

Comment:
Rolling doesn't seem to actually do anything when set like this, I think we can safely omit this. The config file lets you change whether the sessions are rolling.